pkg/status/status.go

```diff
@@ -332,11 +332,12 @@ func (s *status) monitorClusterStatus() {
 		select {
 		case <-s.stopCh:
 			// If the stopCh is closed, set all ClusterOperatorStatus conditions to false.
+			reason := "operatorExited"
```

Comment:
Camel case is `OperatorExited`. Please fix everywhere.

Reply:
Will do.

Reply:
Interesting side note, the common examples [provided via wikipedia ](https://en.wikipedia.org/wiki/Camel_case) include `iPhone` and `eBay`, and the format that starts with a capital letter is typically called `UpperCamelCase` or `PascalCase`.

Existing reasons in `ClusterOperatorStatus Conditions` other Reason are using `PascalCase` so I suppose we should too.

Reply:
Interesting indeed. Lets be consistent with existing reasons.